pykp/model.py

```diff
@@ -566,7 +589,8 @@ def decode(self, trg_inputs, src_map, oov_list, enc_context, enc_hidden, trg_mas
         (2) Standard Attention
         '''
         # Get the h_tilde (batch_size, trg_len, trg_hidden_dim) and attention weights (batch_size, trg_len, src_len)
-        h_tildes, attn_weights, attn_logits = self.attention_layer(decoder_outputs.permute(1, 0, 2), enc_context, encoder_mask=ctx_mask)
+        sep = [(t == 4).nonzero().squeeze() for t in trg_inputs]
```

Comment:
add `self.sep_word = opt.word2id[pykp.io.SEP_WORD]` in __init__(), and use `self.sep_word` here